### DISCUSSION TITLE
Grain yield is less affected by spike no in APSIMX Barley/Wheat

### DISCUSSION THREAD

**Discussion by KeLiu7** [APSIMInitiative/ApsimX#9370] [Q&A] [answered]
I found barley grain yield is not that sensitive to spike No in APSIMX. For example, when Spike No was equal to 820, the corresponding yield was 9413kg/ha, and when I tried to reduce this number to 749 by increasing the BranchMortality, the yield only dropped by 0.1%. 
Does the calculation of grain yield in APSIMX have nothing to do with Spike No?

As an important part of the yield composition, this is far from the actual situation. Could you please help to explain this? Many thanks @hut104 @HamishBrownPFR

**Comment by hut104** [comment, marked as answer]
The model does not use spike number to determine grain number

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: KeLiu7 ***@***.***>
Sent: Thursday, October 3, 2024 5:19:47 PM
To: APSIMInitiative/ApsimX ***@***.***>
Cc: Huth, Neil (A&F, Toowoomba) ***@***.***>; Mention ***@***.***>
Subject: [APSIMInitiative/ApsimX] Grain yield is less affected by spike no in APSIMX Barley/Wheat (Discussion #9370)


I found barley grain yield is not that sensitive to spike No in APSIMX. For example, when Spike No was equal to 820, the corresponding yield was 9413kg/ha, and when I tried to reduce this number to 749 by increasing the BranchMortality, the yield only dropped by 0.1%.
Does the calculation of grain yield in APSIMX have nothing to do with Spike No?

As an important part of the yield composition, this is far from the actual situation. Could you please help to explain this? Many thanks @hut104<https://github.com/hut104> @HamishBrownPFR<https://github.com/HamishBrownPFR>
